### PR TITLE
docs(relnote): Fx114 - crossorigin attribute for image and feImage elements

### PR DESCRIPTION
**Pull request by bsmth**
Fx 114 supports `crossorigin` attribute on `<image>` and `<feimage>` elements which allow for specifying client behavior for cross-origin resource requests.

```html
<svg width="200" height="200" xmlns="http://www.w3.org/2000/svg">
  <image
    href="https://example.com/mdn_logo_dark.png"
    height="200"
    width="200"
    crossorigin="use-credentials" />
</svg>
```
__Related issues and pull requests:__
- [ ] Parent issue https://github.com/mdn/content/issues/26690
- [x] content: https://github.com/mdn/content/pull/27065

__Bugzilla:__
- [BUG-1240357](https://bugzilla.mozilla.org/show_bug.cgi?id=1240357)

__Resources:__
* WPT: https://wpt.live/svg/embedded/image-crossorigin.sub.html
* WPT overview: https://wpt.fyi/results/svg/embedded/image-crossorigin.sub.html?label=master&product=chrome%5Bstable%5D&product=edge%5Bstable%5D&product=firefox%5Bexperimental%5D&product=safari%5Bstable%5D&aligned
* specs
  * https://svgwg.org/svg2-draft/embedded.html#ImageElementCrossoriginAttribute
  * https://drafts.fxtf.org/filter-effects/#element-attrdef-feimage-crossorigin

### CODE REVIEW

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Can you also add `crossorigin` to https://github.com/mdn/browser-compat-data/blob/main/api/SVGImageElement.json and https://github.com/mdn/browser-compat-data/blob/main/api/SVGFEImageElement.json.

**Comment by bsmth** [comment]
> Can you also add `crossorigin` to https://github.com/mdn/browser-compat-data/blob/main/api/SVGImageElement.json and https://github.com/mdn/browser-compat-data/blob/main/api/SVGFEImageElement.json. 

Will do 👍

File: api/SVGImageElement.json [hunk 1]
```diff
@@ -43,6 +43,39 @@
           "deprecated": false
         }
       },
+      "crossOrigin": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGImageElement/crossorigin",
```
Comment (Elchi3) [inline api/SVGImageElement.json:48]:
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGImageElement/crossorigin",
          "spec_url": "https://svgwg.org/svg2-draft/embedded.html#__svg__SVGImageElement__crossOrigin"
```

**Comment by bsmth** [comment]
Thanks @Elchi3 - applied your suggestion if you want to have a look 👀

**Comment by bsmth** [comment]
Danke dir 🚀

**Pull request merged**